### PR TITLE
Adding buildscript for package starlark-go

### PR DESCRIPTION
**Pull request by vathsalashetty25**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@kumarvikas2020 please review

File: s/starlark-go/starlark-go_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+wget https://golang.org/dl/go1.17.linux-ppc64le.tar.gz
+rm -rf /home/go && tar -C /home -xzf go1.17.linux-ppc64le.tar.gz
+rm -f go1.17.linux-ppc64le.tar.gz
+export GOPATH=/home/go
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+export  GO111MODULE=on
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
```
Comment (kumarvikas2020) [inline s/starlark-go/starlark-go_ubi_8.5.sh:39]: You may want to add a check for the git clone command.

**Comment by vathsalashetty25** [comment]
Script updated with necessary changes.

**Comment by vathsalashetty25** [comment]
Closing PR to re-open again

**Comment by seth-priya** [comment]
@kumarvikas2020 please confirm changes

File: s/starlark-go/starlark-go_ubi_8.5.sh (hunk 1 above)
Comment (kumarvikas2020) [reply s/starlark-go/starlark-go_ubi_8.5.sh:41]: @vathsalashetty25 Any update on this?

Comment (vathsalashetty25) [reply s/starlark-go/starlark-go_ubi_8.5.sh:41]: Yes..Instead of Patch file,i have used sed command...it's working same as patchfile

Comment (vathsalashetty25) [reply s/starlark-go/starlark-go_ubi_8.5.sh:41]: yes..i have added check for git clone command

**Review by kumarvikas2020 — APPROVED** [review]

**Pull request merged**